### ISSUE TITLE
Mozilla News Beat: July 10, 2020

### ISSUE THREAD

**Issue by sabrinang** [MozillaFoundation/Design#600]
### Intro

Create assets for weekly News Beat (social, email, blog). Content TBD on Thursday to be published Thursday (as we have Friday off).

**Comment by sabrinang** [comment]
Redpen for News Beat July 10 here: https://redpen.io/p/fgdf65487ab380dd19

Graphic assets are exported here in gdrive: https://drive.google.com/drive/folders/1xOi1jptSX89DJWsOeCOSrldfGhNmxOx9?usp=sharing

Currently in review, waiting for approval.

During our meeting we also discussed iterating something that featured top story, gif/video to attract more views on instagram stories (as we have a same post weekly). We also discussed changing colour themes again. Since this is a quick turn around, I will most likely explore this for next week as it isn't urgent for this week.
